chrome/background.js

```diff
@@ -2,28 +2,28 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-var prefix = 'https://webcompat.com/?open=1&url=';
-var screenshotData = '';
+var prefix = `https://webcompat.com/?open=1&url=`;
+var screenshotData = ``;
 
 chrome.contextMenus.create({
-  id: "webcompat-contextmenu",
-  title: "Report site issue",
-  contexts: ["all"]
+  id: `webcompat-contextmenu`,
+  title: `Report site issue`,
+  contexts: [`all`]
 });
 
 
-function reportIssue (tab) {
-  chrome.tabs.captureVisibleTab({format: 'png'}, function(res) {
+function reportIssue() {
```

Comment:
(I'm guessing maybe eslint complains? It's nice to know that a tab object is being passed into `reportIssue`, IMO.)